x-pack/plugins/security_solution/public/common/components/header_section/index.tsx

```diff
@@ -117,76 +114,85 @@ const HeaderSectionComponent: React.FC<HeaderSectionProps> = ({
                   <EuiFlexItem grow={false}>
-                    <EuiButtonIcon
-                      data-test-subj="query-toggle-header"
-                      aria-label={i18n.QUERY_BUTTON_TITLE(toggleStatus)}
-                      color="text"
-                      display="empty"
-                      iconType={toggleStatus ? 'arrowDown' : 'arrowRight'}
-                      onClick={toggle}
-                      size="s"
-                      title={i18n.QUERY_BUTTON_TITLE(toggleStatus)}
+                    <InspectButton
+                      isDisabled={isInspectDisabled}
+                      queryId={id}
+                      multiple={inspectMultiple}
+                      title={title}
                     />
                   </EuiFlexItem>
                 )}
-                <EuiFlexItem>
-                  <EuiTitle size={titleSize}>
-                    <h4 data-test-subj="header-section-title">
-                      <span className="eui-textBreakNormal">{title}</span>
-                      {tooltip && (
-                        <>
-                          {' '}
-                          <EuiIconTip color="subdued" content={tooltip} size="l" type="iInCircle" />
-                        </>
-                      )}
-                    </h4>
-                  </EuiTitle>
-                </EuiFlexItem>
-              </EuiFlexGroup>
 
-              {!hideSubtitle && toggleStatus && (
-                <Subtitle data-test-subj="header-section-subtitle" items={subtitle} />
-              )}
+                {headerFilters && toggleStatus && (
+                  <EuiFlexItem data-test-subj="header-section-filters" grow={false}>
+                    {headerFilters}
+                  </EuiFlexItem>
+                )}
+              </EuiFlexGroup>
             </EuiFlexItem>
 
-            {id && showInspectButton && toggleStatus && (
-              <EuiFlexItem grow={false}>
-                <InspectButton
-                  isDisabled={isInspectDisabled}
-                  queryId={id}
-                  multiple={inspectMultiple}
-                  title={title}
-                />
-              </EuiFlexItem>
-            )}
-
-            {headerFilters && toggleStatus && (
-              <EuiFlexItem data-test-subj="header-section-filters" grow={false}>
-                {headerFilters}
+            {children && toggleStatus && (
+              <EuiFlexItem data-test-subj="header-section-supplements" grow={split ? true : false}>
+                {children}
               </EuiFlexItem>
             )}
           </EuiFlexGroup>
         </EuiFlexItem>
-
-        {children && toggleStatus && (
-          <EuiFlexItem data-test-subj="header-section-supplements" grow={split ? true : false}>
-            {children}
+        {!hideSubtitle && toggleStatus && (
+          <EuiFlexItem>
+            <Subtitle data-test-subj="header-section-subtitle" items={subtitle} />
```

Comment:
All of the jsx changes here are in service of this fix:
Before:
<img width="706" alt="Screen Shot 2022-05-10 at 10 23 19 AM" src="https://user-images.githubusercontent.com/6935300/167676861-8bdd52dc-8b69-4744-9359-227b76e6c3a3.png">
After:
<img width="698" alt="Screen Shot 2022-05-10 at 10 23 27 AM" src="https://user-images.githubusercontent.com/6935300/167676849-a74de07a-bde6-4586-85ab-831d5f813f74.png">